index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+    
+    <div id="title">
+      <h1> Dr. Norman Borlaug </h1>
+      <p>The man who saved a billion lives</p>
+    </div>
+
+    
+
+    
```

Comment:
Please use two spaces as an indentation in the HTML and CSS instead of the tab (4 spaces) because when multiple people are working on the same project, the code would have a consistent indentation.
Please use the blank lines only for the logical separation of elements. This is again for consistency when multiple developers are contributing and readability.

Reply:
Great job telling the code writer what needs to be changed and the reasons for the suggested changes 👌. I think you can also include emojis ☺ and gifs to make the code review more human-friendly😎

Reply:
@Lizdev-05 is it okay to block code review for not using emojis?  🥲

Reply:
Good day, @Anand8317, I did not block your code, I just requested changes. As simple as adding emojis are, it is also important in your review☺

Reply:
I am sorry about that. I noticed now that it is my mistake, I should have marked it as optional.